posydon/popsyn/population_params_default.ini

```diff
@@ -205,37 +205,39 @@
     # builtin posydon step
   absolute_import = None
     # 'package' kwarg for importlib.import_module
-  mechanism = 'Patton&Sukhbold20-engine'
-    # 'direct', Fryer+12-rapid', 'Fryer+12-delayed',
-    # 'Sukhbold+16-engine', 'Patton&Sukhbold20-engine'
-  engine = 'N20'
-    # 'N20' for 'Sukhbold+16-engine',
-    # 'Patton&Sukhbold20-engine' or '' for the others
-  PISN = "Marchant+19"
+  mechanism = 'Fryer+12-delayed'
+    # in v2: 'Fryer+12-rapid', 'Fryer+12-delayed', 'Patton&Sukhbold20-engine'
+    # need profiles: 'direct', 'Sukhbold+16-engine'
+  engine = ''
+    # 'N20' for 'Sukhbold+16-engine', 'Patton&Sukhbold20-engine'
+    # or '' for the others
+  PISN = "Hendriks+23"
     # None, "Marchant+19", "Hendriks+23"
   PISN_CO_shift = 0.0
-    # float (-inf,inf)
+    # float (-inf,inf) in v2 only 0.0
```

Comment:
It should be made clear that these values are only applicable when using `interp_values=True` in v2.
It's not for the whole of v2. It is still possible to use other values in v2 and still use the interpolator in the stellar evolution. You will just miss the details of the compact objects, but that might be fine for some others.